RedHatInsights/insights-advisor-frontend
# feat(RHIF-158): adopt new pdf-generator service

# Description

Associated Jira ticket: # [(RHIF-158)](https://issues.redhat.com/browse/RHIF-158)

Adopts the new pdf-generator service. The executive report has been changed to use the new service instead of the shared library. The pdf report on the systems page is removed completely according to the PM's [request](https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1686833724295099).

# How to test the PR

1.  Run the PR locally
2. Navigate to recommendations page
3. click on 'Download executive report' link on the top right.
4. Observe that pdf report gets loaded, notifications are in place. Filename is as expected
5. Navigate to systems page from the navigations sidebar
6. observe that systems table does not have pdf export option available on the toolbar export options 

# Before the change


# After the change


# Dependent work link


# Checklist:
